File: android/src/main/java/com/example/plugin/BackgroundChangerPlugin.java
package com.example.plugin.backgroundchanger;

import android.graphics.Color;

import com.getcapacitor.Plugin;
import com.getcapacitor.PluginCall;
import com.getcapacitor.PluginMethod;
import com.getcapacitor.annotation.CapacitorPlugin;

@CapacitorPlugin(name = "BackgroundChanger")
public class BackgroundChangerPlugin extends Plugin {

    @PluginMethod
    public void changeBackgroundToTransparent(PluginCall call) {
        getBridge().executeOnMainThread(() -> {
            getBridge().getWebView().setBackgroundColor(Color.TRANSPARENT);
            call.resolve();
        });
    }

    @PluginMethod
    public void changeBackgroundToWhite(PluginCall call) {
        getBridge().executeOnMainThread(() -> {
            getBridge().getWebView().setBackgroundColor(Color.WHITE);
            call.resolve();
        });
    }
}

File: android/src/main/java/com/example/BackgroundChangerPlugin.kt
package com.example.plugin

import com.getcapacitor.JSObject
import com.getcapacitor.Plugin
import com.getcapacitor.PluginCall
import com.getcapacitor.PluginMethod
import com.getcapacitor.annotation.CapacitorPlugin

@CapacitorPlugin(name = "BackgroundChanger")
class BackgroundChangerPlugin : Plugin() {

    @PluginMethod
    fun changeBackground(call: PluginCall) {
        activity.runOnUiThread {
            bridge.webView.setBackgroundColor(Color.TRANSPARENT)
        }
        
        call.resolve()
    }
}